index.js

```diff
@@ -9,6 +9,22 @@ function isBoolean(obj) { return toS(obj) === '[object Boolean]'; }
 function isNumber(obj) { return toS(obj) === '[object Number]'; }
 function isString(obj) { return toS(obj) === '[object String]'; }
 
+var TypedArrays = [
+	'[object Int8Array]',
+	'[object Uint8Array]',
+	'[object Uint8ClampedArray]',
+	'[object Int16Array]',
+	'[object Uint16Array]',
+	'[object Int32Array]',
+	'[object Uint32Array]',
+	'[object Float32Array]',
+	'[object Float64Array]',
+];
+
+function isTypedArray(obj) {
+	return TypedArrays.indexOf(Object.prototype.toString.call(obj)) >= 0;
+}
```

Comment:
this isn't actually robust; https://npmjs.com/is-typed-array is the only reliable approach.